packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -1,21 +1,51 @@
 import featureManagement from '../src/index';
-import expectations from './data/e2e-expectations.json';
+import { setContextGetter } from '../src/contextGetter';
+import trackExperiment from '../src/trackExperiment';
 
-const { getBooleanValue, getIntegerValue, getStringValue } = featureManagement('je-web-core');
+const expectations = require('./data/e2e-expectations.json');
+const config = require('./data/e2e-fmconfig-mixed.json');
 
-const functionMap = { boolean: getBooleanValue, integer: getIntegerValue, string: getStringValue };
+const logger = {
+    logError(){},
+    logWarn(){},
+    logInfo(){}
+}
+
+const fm = featureManagement({
+    json: JSON.stringify(config),
+    logger: logger
+});
+
+const functionMap = { bool: fm.getBooleanValue.bind(fm), int: fm.getIntegerValue.bind(fm), string: fm.getStringValue.bind(fm) };
```

Comment:
As you've done it, the answer is probably "yes", but is the `bind`ing necessary here?

Reply:
It's because the functions get pulled from their context (which is methods on a class) into this "map".  So when you run it `this` is undefined.  It might be that a) there is a better way of enforcing that they are bound somehow or b) we should avoid classes and this's.  But I think we can work through that as we start to iterate on it and integrate it with consumers.